src/raudio.c

```diff
@@ -1727,19 +1775,10 @@ void UnloadMusicStream(Music music)
     }
 }
 
-// Start music playing (open stream)
+// Start music playing (open stream) from beginning
 void PlayMusicStream(Music music)
 {
-    if (music.stream.buffer != NULL)
-    {
-        // For music streams, we need to make sure we maintain the frame cursor position
-        // This is a hack for this section of code in UpdateMusicStream()
-        // NOTE: In case window is minimized, music stream is stopped, just make sure to
-        // play again on window restore: if (IsMusicStreamPlaying(music)) PlayMusicStream(music);
-        ma_uint32 frameCursorPos = music.stream.buffer->frameCursorPos;
-        PlayAudioStream(music.stream);  // WARNING: This resets the cursor position.
-        music.stream.buffer->frameCursorPos = frameCursorPos;
-    }
+    PlayAudioStream(music.stream);
```

Comment:
Could this be simplified this way?

Reply:
I'm sorry, I don't understand what you mean.
If you are generally asking whether this change is necessary and reduces this logic to just the single call, the reasoning is described in this comment: https://github.com/raysan5/raylib/issues/3664#issuecomment-2044458367

Long story short: The hack, in overall, is not necessary as it was a no-op for the described use-case.
The "only" functional difference is now that `PlayMusicStream()` is no longer keeping the current position that it had. Though, my understanding is that this is not wanted anyway.

Reply:
I see, thanks!